OTVision/detect/detect.py

```diff
@@ -140,6 +158,7 @@ def main(
         if debug:
             reset_debug()
         return detections_img_file_chunks, class_labels
+    log.info("")
```

Comment:
Tried to guess, but nothing came to my mind. Better you fill the quotation marks :)

Reply:
When done, please merge :) Rest looks good!

Reply:
I forgot to remove that one!